maintainers/depmgmt.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: default
+nav_order: 4
+parent: RAPIDS Maintainer Docs
+title: Dependency Management
+---
+
+# Dependency Management Documentation
+
+## Overview
+
+Outlines how to test and change dependencies of the RAPIDS projects
+
+### Intended audience
+
+Developers
+{: .label .label-green}
+
+Project Leads
+{: .label .label-blue}
+
+Operations
+{: .label .label-purple}
+
+## Adding a Dependency
+
+The [RAPIDS Integration repository](https://github.com/rapidsai/integration) contains all the information needed for adding a dependency to the RAPIDS projects. The `meta.yaml` you change should be based on the purpose of the package you are installing. The purpose of each meta-package is listed below:
```

Comment:
I think it's a good practice to avoid first and second point of view for technical docs. I'd suggest removing `you` here and replacing it with something like:

_The `meta.yaml` changed should be based on the purpose of the package to be installed._